build.cmd

```diff
@@ -6,15 +6,24 @@ setlocal
 ::       means that that rebuilding cannot successfully delete the task
 ::       assembly. 
 
-:: Check prerequisites
-if not defined VS120COMNTOOLS (
-    if not defined VS140COMNTOOLS (
-        echo Error: build.cmd should be run from a Visual Studio 2013 or 2015 Command Prompt.  
-        echo        Please see https://github.com/dotnet/corefx/wiki/Developer-Guide for build instructions.
-        exit /b 1
+if not defined VisualStudioVersion (
+    if defined VS140COMNTOOLS (
```

Comment:
@weshaggard Can decide the policy.  Note that you can always explicitly boot into whichever environment you want to force a specific version.